docs/administration/managing-users-and-teams/service-accounts.md

```diff
@@ -28,12 +28,19 @@ Creating a new Service account is very similar to creating a new User account:
 2. Check **The user is a service account** to indicate this will be a Service account
 3. Enter a unique **Username** and **Display name** so you can distinguish this Service account
 4. Click the **Create user** button to create the Service account
-![](/docs/images/3049520/3278574.png "width=500")
+
+![Create service account](create-svc.png)
 
 :::hint
 This Service account is not very useful until it [belongs to one or more teams](/docs/administration/managing-users-and-teams/index.md), and has one or more [Octopus API keys](/docs/how-to/how-to-create-an-api-key.md) associated with it
 :::
 
-![](/docs/images/3049520/3278575.png "width=500")
```

Comment:
Should we delete this image file as well?

Reply:
yep, meant to do that, good catch!